api/src/main/java/org/openmrs/api/ConditionClinicalStatus.java

```diff
@@ -0,0 +1,29 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+
+package org.openmrs.api;
+
+/**
+ * The ConditionClinicalStatus is what we've historically called "condition status" and what FHIR refers to as
+ * ConditionClinicalStatus . This is a clinical condition that has risen to a level of concern.
+ *
+ * ACTIVE - This is where the subject is currently experiencing the symptoms of the condition or there is evidence of the condition.
```

Comment:
Can you describe the enum values right next to each, instead of in the class description?

Reply:
Alright. On it.

Reply:
Done

Reply:
@dkayiwa Have refactored comments and added javadoc to the method